etc/profile-m-z/tidal-hifi.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for tidal-hifi
+# Description: The web version of Tidal running in electron with hifi support thanks to widevine.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tidal-hifi.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}.cache/AMD/VkCache
+noblacklist ${HOME}.cache/fontconfig
+noblacklist ${HOME}.cache/mesa_shader_cache
+noblacklist ${HOME}.config/Electron
+noblacklist ${HOME}.config/pulse
+noblacklist ${HOME}.config/tidal-hifi
```

Comment:
Never blacklisted (except of Electron).

Reply:
I made my noblacklist/whitelist/mkdir's based on the comment in the template suggesting to ls -aR in the sandbox, I'll adjust this to just `.config/tidal-hifi` since I think that's the only one that actually seems necessary in my testing.